modules/document_repository/js/document_repository_helper.js

```diff
@@ -195,6 +195,7 @@ function uploadForm() {
 function renderTree() {
   var fileDir = JSON.parse($("#json_data").html());
   var filtered = JSON.parse($("#isFiltered").html()).filtered;
+  var hasDeletePerm = JSON.parse($("#hasDeletePerm").html()).hasDeletePerm;
```

Comment:
this seems like a bad idea. can you just use `loris.userHasPermission('document_repository_delete')` ?

Reply:
did not know we could do that is js!